Translation correction (#109)
07th-mod/higurashi-console-arcs: Update/some_009.txt

//r親友としての、共感か。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "親友としての、共感か。",
			NULL, "Empathy as a close friend?", Line_Normal);
	ClearMessage();

//r人間としての、同情か。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "人間としての、同情か。",
			NULL, "Sympathy as a person?", Line_Normal);
	ClearMessage();

//rそれとも、…憎むべき相手と同じ出身者としての、良心の呵責だろうか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それとも、…憎むべき相手と同じ出身者としての、良心の呵責だろうか。",
			NULL, "Or... pangs of conscience as someone from the very same place as the one I should hate?", Line_Normal);
	ClearMessage();

//rわからない。…私は、今の私が、わからない…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "わからない。…私は、今の私が、わからない…。",

> 			NULL, "I don't know... I don't know what feel anymore...", Line_Normal);

Reviewer: ```suggestion
			NULL, "I don't know... I don't know what to feel anymore...", Line_Normal);
```

//rぎゅっ、と私の手を握りながら、千紗登ちゃんは私の顔を見る。…『断ろうか？』と小さな声で囁いたが、私はゆっくりとかぶりを振ってから、こくん、と頷いた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ぎゅっ、と私の手を握りながら、千紗登ちゃんは私の顔を見る。…『断ろうか？』と小さな声で囁いたが、私はゆっくりとかぶりを振ってから、こくん、と頷いた。",

> 			NULL, "Chisato-chan gripped my hands tight and looked into my eyes. ...She whispered \"Should I refuse?\" to me, but I slowly shook my head, and then nodded.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Chisato-chan gripped my hands tight and looked into my eyes. ...She whispered \"Should I turn them away?\" to me, but I slowly shook my head, and then nodded.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s04/28/310500223", 256, TRUE);
	OutputLine(NULL, "「失礼します——」",
			NULL, "\"Pardon the intrusion...\"", Line_Normal);
	ClearMessage();

//rそして、潜めた声の少し後でがちゃり、とノブの音を立てて、ドアが開く。その向こうには、昼間と同じ赤いスーツを着た南井刑事が硬い表情で立っていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、潜めた声の少し後でがちゃり、とノブの音を立てて、ドアが開く。その向こうには、昼間と同じ赤いスーツを着た南井刑事が硬い表情で立っていた。",

> 			NULL, "And just a moment later, I heard the rattling of the doorknob as the door opened. Standing the other side with a firm expression was Detective Minai, wearing the same red suit from before.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And just a moment later, I heard the rattling of the doorknob as the door opened. Standing on the other side with a firm expression was Detective Minai, wearing the same red suit from before.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300513", 256, TRUE);
	OutputLine(NULL, "「あ、あんたねぇっ！",
			NULL, "\"Y-you've got some nerve!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300514", 256, TRUE);
	OutputLine(NULL, "　夏美がさっきどういう思いをしたのかわかってんのか？！」",

> 			NULL, " Do you have any idea how Natsumi was feeling just a minutte ago?!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Do you have any idea how Natsumi was feeling just a minute ago?!\"", Line_Normal);
```

//r激昂して掴みかかろうとするその腕を、私は寸前で抑える。そして少しの間千紗登ちゃんを見つめてから、南井刑事に目を向けて言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "激昂して掴みかかろうとするその腕を、私は寸前で抑える。そして少しの間千紗登ちゃんを見つめてから、南井刑事に目を向けて言った。",

> 			NULL, "I grabbed Chisato-chan's arm just as she was about to lunge at Detective Minai with fury. I looked at her for a few seconds after, then turned toward Detective Minai and began to speak.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I grabbed Chisato-chan's arm just as she was about to lunge at Detective Minai with fury. I looked at her for a few seconds, then turned toward Detective Minai and began to speak.", Line_Normal);
```

//r…千紗登ちゃんはしばらくどうすべきかと逡巡していたけど、私の気持ちを汲んでくれたのか結局、『わかった』と同意して席を立ってくれた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…千紗登ちゃんはしばらくどうすべきかと逡巡していたけど、私の気持ちを汲んでくれたのか結局、『わかった』と同意して席を立ってくれた。",
			NULL, "...Chisato-chan hesitated for a while, but eventually agreed and stood up, perhaps out of consideration for my wishes.", Line_Normal);
	ClearMessage();

	FadeBustshot( 3, FALSE, 0, 0, 0, 0, 200, TRUE );

//r——ドアを開けて部屋を出て行く間際、南井刑事をきつく睨み付けて…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "——ドアを開けて部屋を出て行く間際、南井刑事をきつく睨み付けて…。",

> 			NULL, "...But just before she opened the door and left the room, she shot daggers at Detective Minai.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...But just before she opened the door and left the room, she stared daggers at Detective Minai.", Line_Normal);
```
